doc/builders/testers.chapter.md

```diff
@@ -14,19 +14,25 @@ for example when using an 'old' hash in a fixed-output derivation.
 Examples:
 
 ```nix
-passthru.tests.version = testVersion { package = hello; };
+passthru.tests.version = testers.testVersion { package = hello; };
 
-passthru.tests.version = testVersion {
+passthru.tests.version = testers.testVersion {
   package = seaweedfs;
   command = "weed version";
 };
 
-passthru.tests.version = testVersion {
+passthru.tests.version = testers.testVersion {
   package = key;
   command = "KeY --help";
   # Wrong '2.5' version in the code. Drop on next version.
   version = "2.5";
 };
+
+passthru.tests.version = testers.testVersion {
+  package = ghr;
+  # The output needs to match 'version' exactly.
```

Comment:
```suggestion
  # The output needs to contain the version string exactly without prefix.
```

Reply:
https://github.com/NixOS/nixpkgs/blob/a23fbeb6e8d5c7f35c9feba25da0061ebcf79efe/pkgs/build-support/testers/default.nix#L12

The `grep -Fw` means, it searches for a word, with whitespace in front and end. It fails when it has "v1.0", so we have to add the "v" here. The whole output line does not have to match.

Feel free to suggest better comment text. My suggestion is a bit confusing.

Reply:
> The grep -Fw means, it searches for a word, with whitespace in front and end. It fails when it has "v1.0", so we have to add the "v" here. The whole output line does not have to match.

Ah, I wasn't aware of that. Thanks for letting me know :+1: